content/docs/Reference/tokens.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "Tokens"
+linkTitle: "Tokens"
+date: 2020-04-03
+weight: 5
+description: >
+  Various tokens that can be used in the commands you send to Cue
+---
+
+Tokens can be used in the commands when submitting outline jobs, either in the graphical Cuesubmit
+client, or via the PyOutline API.
+
+Following are the commonly used tokens:
```

Comment:
I recommend updating to:

This page describes tokens you can use in commands when submitting OpenCue jobs, either in the graphical Cuesubmit
client, or using the PyOutline API.

(I would delete the sentence 'Following are the commonly used tokens:' as we're not 
introducing a list)